lib/internal/cluster/master.js

```diff
@@ -67,19 +68,15 @@ cluster.setupMaster = function(options) {
 
   initialized = true;
   schedulingPolicy = cluster.schedulingPolicy;  // Freeze policy.
-  assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
-         `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
-
-  const hasDebugArg = process.execArgv.some((argv) => {
-    return /^(--debug|--debug-brk)(=\d+)?$/.test(argv);
-  });
+  if (schedulingPolicy !== SCHED_NONE && schedulingPolicy !== SCHED_RR)
```

Comment:
this is arguably easier to read than the original, because it doesn't break function args across lines, but since we know what assert is just `function assert(x) { if(!x) ...`, its weird to pull code out of a function, functions are there to re-use code. Are you saying that inlining conditionals and avoiding function calls is a big perf boost? assert/`function ok()` is so short, I'd think it would be inlineable by v8.